nodejs-bundle-webpack/nodejs-bundle-webpack.spec

```diff
@@ -0,0 +1,191 @@
+%global npm_name webpack
+
+Name: nodejs-%{npm_name}
+Version: 1.13.1
+Release: 1%{?dist}
+Summary: Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jade, coffee, css, less, ... and your custom stuff.
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/webpack/webpack.git
+Source0: http://registry.npmjs.org/webpack/-/webpack-1.13.1.tgz
```

Comment:
All other bundled packages inside this should be listed here as sources too, then they can all be installed in %prep, %install etc.